src/dateutils.ts

```diff
@@ -1,5 +1,7 @@
-const XDate = require('xdate');
-const {toMarkingFormat} = require('./interface');
+import XDateClass from 'xdate';
+import {toMarkingFormat} from './interface';
+
+const XDate = XDateClass as any;
```

Comment:
`require` types modules as `any`, since this change is leading to a lot of typing problems, I prefered to keep the same behavior as before